src/ui/components/UpdateDialog/index.tsx

```
@@ -89,17 +89,18 @@ export const UpdateDialog = () => {
       </Box>
 
       <ButtonGroup>
-        <Button type='button' onClick={handleSkipButtonClick}>
+        <Button type='button' onClick={handleSkipButtonClick} aria-label={t('dialog.update.skip')} >
```

ESLint

[error] Replace `·type='button'·onClick={handleSkipButtonClick}·aria-label={t('dialog.update.skip')}` with `⏎··········type='button'⏎··········onClick={handleSkipButtonClick}⏎··········aria-label={t('dialog.update.skip')}⏎·······`

(prettier/prettier)